ipatests/test_webui/ui_driver.py

```diff
@@ -2007,6 +2007,43 @@ def run_cmd_on_ui_host(self, cmd):
             cmd=cmd
         )
 
+    def add_user_auth_type(self, auth_type, save=False):
+        """
+        Select user auth type
+        :param auth_type: one of password, radius, otp, pkinit or hardened
+        """
+        s_checkbox = 'div[name="ipauserauthtype"] input[value="{}"]'.format(
+            auth_type)
+        checkbox = self.find(s_checkbox, By.CSS_SELECTOR, strict=True)
+        if not checkbox.is_selected():
+            checkbox.click()
+        if save:
+            self.facet_button_click('save')
+
+    def remove_user_auth_type(self, auth_type, save=False):
```

Comment:
I don't any usage of the method.

Reply:
there is none, but as all similar methods have add/assert/remove methods, I implemented for the future

Reply:
You are not obligated to implement all CRUD operations here.

It will be hard to maintain such redundant code. So I would suggest to remove it.

Reply:
As you wish, removed.